TileStache/Goodies/VecTiles/transform.py

```diff
@@ -137,6 +137,8 @@ def normalize_osm_id(shape, properties, fid):
         binary = wkb.dumps(shape)
         fid = md5.new(binary).hexdigest()[:10]
     else:
+        # always include an osm_id in the properties map
+        properties['osm_id'] = fid
```

Comment:
Good point; it's not. Sometimes it's an id from a natural earth dataset.

What do you think is the best way to handle that? I think we'll have to signal that somehow from the queries themselves. Some options:

1. some queries include a `source`. We can ensure that all queries that return back ids that aren't from osm include this field. The transform can then check to see if the source isn't openstreetmap before including the `osm_id` property.
2. for all osm queries, we can include a column in the queries separate from the `id` called `osm_id`. Then we can have a separate transform that just checks for the presence of the `osm_id`, and makes it the absolute value (and might as well move the `osm_relation` logic there too).